anilist/src/main/java/com/huchihaitachi/anilist/presentation/AnilistController.kt

```diff
@@ -160,34 +185,55 @@ class AnilistController : Controller(), AnilistView {
     )
   }
 
+  private fun setupContentOverlay() {
+    binding.animeListL.dimOverlayView.setOnClickListener {
+      _hideDetails.onNext(Unit)
+    }
+  }
+
   private fun bindDetailsData(details: Anime) {
-    binding.detailsL.let { detailsBinding ->
-      details.coverImage?.let(detailsBinding.detailsCoverIv::setUrl)
-      details.title?.let { title ->
-        detailsBinding.titleTv.text = resources?.getString(
+    resources?.let { res ->
+      binding.detailsL.let { detailsBinding ->
+        detailsBinding.titleTv.setTextAndHighlight(
+          res,
+          spanFactory.createBold(),
           R.string.title,
-          title,
-          details.type?.stringRes?.let { resources?.getString(it) }
+          R.string.highlighted_title,
+          details.title,
+          (details.type ?: Type.ANIME).stringRes.let(res::getString)
         )
-      }
-      detailsBinding.beginningTv.text = resources?.getString(
-        R.string.beginning,
-        details.season?.localized(resources),
-        details.seasonYear
-      )
-      details.episodes?.let { episodes ->
-        detailsBinding.episodesTv.text = resources?.getString(R.string.num_episodes, episodes)
-      }
-      details.duration?.let { duration ->
-        (duration / 60).let { hours ->
-          detailsBinding.durationTv.text = if(hours == 0) {
-            resources?.getString(R.string.episode_duration_minutes, duration % 60)
-          } else {
-            resources?.getString(R.string.episode_duration_hours, hours, duration % 60)
+        details.coverImage?.let(detailsBinding.detailsCoverIv::setUrl)
+        detailsBinding.episodesTv.setTextAndHighlight(
+          res,
+          spanFactory.createBold(),
+          R.string.num_episodes, R.string.highlighted_num_episodes,
+          details.episodes
+        )
+        details.duration?.let { duration ->
+          (duration / 60).let { hours ->
+            if(hours == 0) {
+              detailsBinding.durationTv.setTextAndHighlight(
+                res,
+                spanFactory.createBold(),
+                R.string.episode_duration_minutes,
+                R.string.highlighted_episode_duration,
+                duration % 60
+              )
+            } else {
+              detailsBinding.durationTv.setTextAndHighlight(
+                res,
+                spanFactory.createBold(),
+                R.string.episode_duration_hours,
+                R.string.highlighted_episode_duration,
+                hours,
+                duration % 60
+              )
+            }
+
           }
         }
+        details.description?.let { description -> detailsBinding.descriptionTv.text = details.description }
```

Comment:
Seems that `details.description` is HTML text
You can use `setText(Html.fromHtml()`